### PR TITLE
leipzig-glossing:0.1.0

### PR DESCRIPTION
**Pull request by neunenak**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

Description: `leipzig-glossing` is a linguistics library for creating interlinear morpheme-by-morpheme glosses according to the [Leipzig glossing rules](https://www.eva.mpg.de/lingua/pdf/Glossing-Rules.pdf).

### CODE REVIEW

File: packages/preview/leipzig-glossing-0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Leipzig Glossing in Typst
+
+`leipzig-glossing` is a [Typst](https://github.com/typst/typst) library for
+creating interlinear morpheme-by-morpheme glosses according to the [Leipzig
+glossing rules](https://www.eva.mpg.de/lingua/pdf/Glossing-Rules.pdf).
+
```
Comment (laurmaedje) [inline packages/preview/leipzig-glossing-0.1.0/README.md:6]: A usage example with an `@preview` import would be great.

File: packages/preview/leipzig-glossing-0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Run `typst compile leipzig-gloss-examples.typ` in the root of the repository to
```
Comment (laurmaedje) [inline packages/preview/leipzig-glossing-0.1.0/README.md:7]: maybe link the repository here?

**Comment by laurmaedje** [comment]
Just a few suggestions. :)

**Comment by laurmaedje** [comment]
I've only recently added this to the guidelines, but example images shouldn't be part of the packages. See also here: https://github.com/typst/packages/pull/17#pullrequestreview-1511235431

**Comment by neunenak** [comment]
Ok I'm going to remove the image for now then. I don't want to permanently host it on github and I need to figure out another way to do that.

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**